### PR TITLE
[Blog] Presto and Alluxio for Hybrid Cloud Data Analytics

### PR DESCRIPTION
**Pull request by madanadit**

### CODE REVIEW

**Comment by brianwarner** [comment]
@aweisberg, can you please take a look at this?

File: website/blog/2020-06-15-alluxio-hybrid-cloud.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+This crucial bottleneck severely limits performance of any workload since a significant portion of its time is spent transferring the requested data between networks that could be residing in geographically disparate locations. As a result, most companies copy their data into a cloud environment and maintain that duplicate data, also known as Lift and Shift. Companies with compliance and data sovereignty requirements may even prevent organizations from copying data into the cloud. This approach is not scalable and requires introducing a lot of manual effort to achieve reasonable results. This article introduces [Alluxio](https://www.alluxio.io/) to serve as a [data orchestration](https://www.alluxio.io/data-orchestration/) layer to help serve data to Presto efficiently, as opposed to either directly querying the distant HDFS cluster or manually providing a localized copy of the data to Presto in a cloud cluster.
+
```
Comment (aweisberg) [inline website/blog/2020-06-15-alluxio-hybrid-cloud.md:10]: Add truncate tag here please

Comment (madanadit) [reply website/blog/2020-06-15-alluxio-hybrid-cloud.md:10]: done

**Comment by madanadit** [comment]
@aweisberg can you help merge this please?

**Comment by aweisberg** [comment]
Can you squash first (and remove the extra commit messages)? @rschlussel  can you merge once that is done?

**Comment by aweisberg** [comment]
Also need to update the date. I tried to update the PR for you, but I don't think maintainers editing is enabled.

**Comment by madanadit** [comment]
@aweisberg Updated the date and squashed commits. @rschlussel Please merge if you're good with this, thank you.

**Pull request merged**